rate: 50

max_linear_speed: 1.1
min_linear_speed: 0.1
max_angular_speed: 2.0
min_angular_speed: 1.0

max_linear_acceleration: 1000000000
max_angular_acceleration: 1000000000

linear_tolerance_outer: 0.3
linear_tolerance_inner: 0.1

angular_tolerance_outer: 0.2
angular_tolerance_inner: 0.1

forwardMovementOnly: False
ignore_angular_tolerance: False

Kp: 1.0
Ka: 6.0
Kb: -0.8

body_width: 0.5969
wheel_diameter: 0.1524

pose_topic: {name: "/global_pose", type: "PoseWithCovarianceStamped"}

odom_topic: "/xvel"
